pmkoalas/complex.py

```diff
@@ -287,6 +303,11 @@ def data(self) -> Mapping[str,object]:
         " returns a data mapping for this collection, i.e. log-level attributes."
         return self._map
     
+    def transition_tree(self) -> 'TransitionTree':
```

Comment:
Not sure I see why every complex event should have this, which is what is implied by a method.

Reply:
fair point, ill drop it from the data structure.